src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -78,7 +79,9 @@ export class HostsComponent implements OnInit {
           host.services.map((service) => {
             service.cdLink = `/perf_counters/${service.type}/${service.id}`;
             const permissionKey = typeToPermissionKey[service.type];
```

Comment:
I would suggest to define the permission here instead of the key.
```suggestion
            const permission = this.permissions[typeToPermissionKey[service.type]];
```
Than your next line would look like this:
```ts
            service.canRead = permission ? permission.read : false;
```

Reply:
Ack. Done.